### PR TITLE
Bulbs Ellipsize

### PR DESCRIPTION
**Pull request by MelizzaP**
truncates text, implemented in the homepage and special coverage players

### CODE REVIEW

**Comment by MelizzaP** [comment]
@collin @daytonn omg, this is finally ready for review

**Comment by MelizzaP** [comment]
[Onion Special Coverage Backlog](https://trello.com/c/YMfvi3i6/667-onion-special-coverage-backlog)

File: elements/bulbs-ellipsize/bulbs-ellipsize.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      'BulbsEllipsize: Expects a line-count attribute');
+    if (!this.textContent) { return; }
+
+    this.truncate();
+  }
+
+  truncate () {
+    let lineCount = parseInt(this.getAttribute('line-count'), 10);
+
+    $(this).truncate({ lines: lineCount, ellipsis: '...' });
```
Comment (daytonn) [inline elements/bulbs-ellipsize/bulbs-ellipsize.js]: I'm not sure we should put jQuery in these elements. Looks like you can do this vanilla js style: [https://www.npmjs.com/package/truncate.js#as-a-vanilla-javascript-object](https://www.npmjs.com/package/truncate.js#as-a-vanilla-javascript-object)

Comment (MelizzaP) [inline elements/bulbs-ellipsize/bulbs-ellipsize.js]: any rational behind that? truncate.js is dependent on jquery already

Comment (daytonn) [inline elements/bulbs-ellipsize/bulbs-ellipsize.js]: The rational is that jQuery is outdated in a code-base like bulbs-elements. There's simply no need to depend on it. I'm not sure what the internals look like for truncate.js but I would bet that it doesn't extend jQuery if it doesn't find it. In our other js code, we're not ready to let go of the almighty `$` but bulbs-elements is stuffed with new hotness. jQuery, in contrast, is old and busted.

Comment (MelizzaP) [inline elements/bulbs-ellipsize/bulbs-ellipsize.js]: while I can always agree with new hotness, there is a bug in the vanilla js version of this. going with jquery.

**Pull request merged**